torchci/pages/api/drci/drci.ts

```diff
@@ -134,6 +135,8 @@ async function addMergeBaseCommits(
     });
 
     pr_info.merge_base = diff.data.merge_base_commit.sha;
+    pr_info.merge_base_date =
+      diff.data.merge_base_commit.commit.committer?.date ?? "";
```

Comment:
I'm surprised to see this needs `commit.committer?.date` instead of `commit.date`.

What logical construct does `committer` refer to? It sounds like a person who did the commit, but is acting more like the act of merging itself.

Reply:
if you look at the output of the api call, commit has a committer and an author
https://stackoverflow.com/questions/18750808/difference-between-author-and-committer-in-git
I don't think it makes a huge difference for us because the bases are usually on main and "authored" and committed at the same time.

Reply:
Thanks for the link! Seems like commit.comitter is indeed the person who did the actual merge, with that date being the date of the merge itself, while commit.author would be the person who wrote the code.

They're usually the same person, but in funky corner cases could be someone different, and the feature was designed for the era when Linus received patches for linux via email!

TIL some new git lore